### ISSUE TITLE
Bug fixes- .NET experience & talk to us changes

### ISSUE THREAD

**Issue by PazYanoverr** [amplication/amplication#8364]

**Comment by PazYanoverr** [comment]
@mulygottlieb  @morhag90 - FYI

**Comment by overbit** [comment]
Test ❌ 

@mulygottlieb the point 1 is not fixed

The rest looks ok
<img width="766" alt="image" src="https://github.com/amplication/amplication/assets/2861984/b8757176-7746-4c71-a433-e383894ac50e">

**Comment by mulygottlieb** [comment]
Yep @overbit my PR was not entirely merged, my bad on statuses. Now it's merged and this is ready for real QA (I already checked and it looks good on sandbox BTW)

**Comment by overbit** [comment]
# Test : ✅
